arangodb-net-standard/Serialization/JsonNetApiClientSerialization.cs

```diff
@@ -83,7 +83,7 @@ public override string SerializeToString<T>(T item, ApiClientSerializationOption
 
             if (serializationOptions.UseCamelCasePropertyNames)
             {
-                jsonSettings.ContractResolver = new CamelCasePropertyNamesExceptDictionaryContractResolver();
+                jsonSettings.ContractResolver = new CamelCasePropertyNamesExceptDictionaryContractResolver(serializationOptions);
```

Comment:
`CamelCasePropertyNamesExceptDictionaryContractResolver` will also affect other aspects of the dictionary serialization appart from camel casing, in which case it shouldn't be in an `if (serializationOptions.UseCamelCasePropertyNames)`, right?

Perhaps we need to review the sequence. I'm a bit confused 🤔

Reply:
@DiscoPYF , 
In `JsonNetApiClientSerialization.SerializeToString()` a `CamelCasePropertyNamesExceptDictionaryContractResolver` is initialized only if `UseCamelCasePropertyNames == true`

In `DictionaryValueConverter.WriteJson()`,  a `CamelCasePropertyNamesExceptDictionaryContractResolver` is initialized and supplied only when `ApplySerializationOptionsToObjectValuesInDictionaries == true` and `UseCamelCasePropertyNames == true` specifically for serialization of dictionaries.